BubbleSort/BubbleSort.Src/Sort.cs

```diff
@@ -0,0 +1,35 @@
+namespace finalTest1;
+
+/// <summary>
+/// Class containing generic bubble sort implementation
+/// </summary>
+public class Sorting
```

Comment:
```suggestion
public static class Sorting
```
В нём только static-методы и вообще нет полей, нет причин быть не static

Reply:
А ещё класс должен называться так же, как файл, в котором он находится